fail2ban/server/database.py

```diff
@@ -34,6 +34,14 @@
 from .ticket import FailTicket
 from ..helpers import getLogger
 
+try:
+	import sqlitebck
```

Comment:
Hmm thinking about it these numbers seem a bit high, I will re-check that soon, probably tomorrow. But in any case they were high enough to immediately draw our attention.